### PR TITLE
Add support for iOS

### PR DESCRIPTION
**Pull request by kawazoe**
This PR strives to adds supports for iOS 11 to iOS 14.5 and beyond.

TODO:
- Update the examples to include an iOS one.
- More in depth testing.

Notes:
I am new to native iOS development, so please criticize harshly 😉

### CODE REVIEW

**Comment by felix-ulonska** [comment]
Is there a reason that this is not merged?

**Comment by kawazoe** [comment]
@felix-ulonska I haven't seen any activity in this project since its creation. You can use my fork if you want. There are also a few more improvements that didn't made it into this PR.

I haven't really maintained it since the newer iOS releases due to lack of hardware so there is a chance it might misbehave. You're open to contribute to my fork if you want, and when the maintainer of this project shows up, I'll happily contribute everything back here.

**Comment by tafelnl** [comment]
I've taken over maintenance of this plugin. My philosophy of approach is a bit different. Allow me to explain myself: on web and iOS the safe area insets are working correctly out of the box. So there's no need patching it for those platforms. On Android however, the safe area insets are not working correctly. Especially in combination with edge-to-edge mode. When edge-to-edge mode is disabled, the safe area insets are supposed to be always anyways. So patching is necessary only on Android, specifically when having edge-to-edge mode enabled.

That's why I've decided to create two methods for this plugin only: `enable` and `disable`. In fact this will enable or disable edge-to-edge mode on Android. It will also patch the safe area insets on Android. On web and iOS the methods have no effect (because remember it's working out of the box).

As a bonus these methods allow you to style the system bars. So you can set them to the color of your liking.

TLDR: I'm closing this PR
